crates/valence_protocol/src/packets/play/update_selected_slot_c2s.rs

```diff
@@ -3,5 +3,5 @@ use super::*;
 #[derive(Copy, Clone, Debug, Encode, Decode, Packet)]
 #[packet(id = packet_id::UPDATE_SELECTED_SLOT_C2S)]
 pub struct UpdateSelectedSlotC2s {
-    pub slot: u8,
+    pub slot: u16,
```

Comment:
This change seems unrelated to this PR. If it's not relevant to boss bars, revert it.

Reply:
Sure I can but it will just bloat the git history for nothing, pr are not shown, so what will be show is that this line has been modify, then later reversed to be rereversed just after. not sure to be really worth it.

Reply:
PRs are squashed merged, so each PR becomes one commit. If we need to revert one, it would be better to not have collateral damage. It'll also make git bisects a bit better.